apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -59,8 +61,15 @@ const useSections = section => {
   return [sections, updateSection];
 };
 
-const saveSection = (e, section) => {
+const saveSection = (e, section, isNewSection) => {
```

Comment:
Adding the isNewSection will help us determine how to save (are we saving a new section or updating an existing one?)